app/javascript/components/widgets/forest-change/glads/index.js

```diff
@@ -20,14 +20,12 @@ export default {
   title: 'Deforestation alerts for the last week in {location}',
   categories: ['forest-change', 'summary'],
   admins: ['adm0', 'adm1', 'adm2'],
-  types: ['geostore', 'wdpa', 'use'],
```

Comment:
I'd bring back the types; the client's request is not about the types per-se but rather the visibility of that widget on the map. The only necessary change here is setting `visible: []` - that way if they change their mind we only need to change the visibility manifest.

Reply:
Agreed here. You just want to leave an empty array if you wish to keep the widget. Leave the visible array as if that is false then a widget always renders. The combo of a valid location, whitelist, and visible array is what allows the widget to be rendered.